src/view/components/toolbar/motion/Accelerometer.tsx

```diff
@@ -44,12 +46,38 @@ interface IProps {
         Z_AXIS: number;
     };
     onUpdateValue: (sensor: SENSOR_LIST, value: number) => void;
+    onSelectGestures?: (event: React.ChangeEvent<HTMLSelectElement>) => void;
+    onSendGesture?: (isActive: boolean) => void;
 }
+
+const GESTURE_BUTTON_MESSAGE = "Send Gesture";
+
 export const Accelerometer: React.FC<IProps> = (props: IProps) => {
     return (
         <div className="AccelerometerBar">
             <br />
             {/* Implement Gestures Here */}
+            <Dropdown options={GESTURES} onSelect={props.onSelectGestures} />
+            <SensorButton
+                label={GESTURE_BUTTON_MESSAGE}
+                onMouseDown={() => {
+                    if (props.onSendGesture) {
+                        props.onSendGesture(true);
+                    }
+                }}

```

Comment:
You can replace this with:
```
onMouseDown={() => props.onSendGesture?.(true)}
```

Reply:
It seems like this functions optional chaining with the typescript version we have is not compatible. This could be added when we update the typescript version. Since this doesn't affect user experience, I will not include this change.

Reply:
function optional chaining is available with TS 3.7.0